app/views.py

```diff
@@ -0,0 +1,152 @@
+from flask import Flask, jsonify, request
+from app.email import Emails
+from app.validator import Validation
+from flasgger import Swagger
+from flasgger import swag_from
+
+
+app = Flask(__name__)
+emails = Emails()
+validator = Validation()
+swagger = Swagger(app)
+
+
+@app.route('/', methods=['GET'])
+def index():
+    """opening route."""
+    return jsonify({
+        "status": 200, 'message': 'welcome to Epic Email.'
+        }), 200
+
+
+@app.route('/api/v1/users/signup', methods=['POST'])
```

Comment:
Please consider changing the URLs for all your endpoints to match those in the document

Reply:
changed all the Urls